internal/data/pg/forms.go

```diff
@@ -51,13 +53,14 @@ func (q *formsQ) Insert(form *data.Form) (string, error) {
 		"phone":     form.Phone,
 		"email":     form.Email,
 		"image":     form.Image,
-	}).Suffix("RETURNING id")
+		"image_url": form.ImageURL,
+	}).Suffix("RETURNING id, nullifier, status, created_at, updated_at")
```

Comment:
RETURNING * doesn't work?

Reply:
The return value can be very large, since the image is a string up to 5 megabytes

Reply:
ok